src/pages/tasks/index.tsx

```diff
@@ -1,3 +1,5 @@
+import { TasksManager } from "@widgets/tasks-manager";
+
 export const TasksPage = () => (
-    <div>Tasks</div>
```

Comment:
// Nitpick
Just add a any title here. It helps to work with `shared` slice, because any `ui` component will be placed there